spec/rubocop/cop/style/rescue_modifier_spec.rb

```diff
@@ -13,6 +13,13 @@
       .to eq(['Avoid using `rescue` in its modifier form.'])
   end
 
+  it 'locates the rescue within the line' do
+    inspect_source(cop,
+                   'method rescue handle')
+    expect(cop.offenses.size).to eq(1)
+    expect(cop.offenses.first.location.source).to eq('rescue')
```

Comment:
Checkout `cop.highlights`. It's the preferred way to check for offence highlights.